source/access/access_using_mobaxterm.rst

```diff
@@ -1,16 +1,25 @@
 .. _access using mobaxterm:
 
-Download and setup MobaXterm
-============================
```

Comment:
removing these will also change the page link in the table of content. It is now sown as prerequisite. So I would keep the 2 lines above, to make clear that this is the MobaXterm page. otherwise it will also not preperly show-up in a search.

Reply:
oops. not sure how that one got lost. 
I used this opportunity to give this page a name that also resembles that of PuTTY page: 'Text-mode access using MobaXterm'